go/vt/wrangler/traffic_switcher.go

```diff
@@ -1168,7 +1168,7 @@ func (ts *trafficSwitcher) createReverseVReplication(ctx context.Context) error
 					// We currently assume the primary vindex is the best way to filter, which may not be true.
 					inKeyrange = fmt.Sprintf(" where in_keyrange(%s, '%s.%s', '%s')", sqlparser.String(vtable.ColumnVindexes[0].Columns[0]), ts.SourceKeyspaceName(), vtable.ColumnVindexes[0].Name, key.KeyRangeString(source.GetShard().KeyRange))
```

Comment:
We don't need to escape these too?

Reply:
`in_keyrange` is analyzed in golang and not seen by mysql, so I believe we are good here.